File: appjava/src/main/java/com/onelogin/mfa/appjava/factors/EmptyFragment.java
package com.onelogin.mfa.appjava.factors;

import android.os.Bundle;

import androidx.fragment.app.Fragment;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.onelogin.mfa.appjava.R;

public class EmptyFragment extends Fragment {

    public EmptyFragment() {
        // Required empty public constructor
    }

    public static EmptyFragment newInstance() {
        return new EmptyFragment();
    }

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        return inflater.inflate(R.layout.fragment_empty, container, false);
    }
}

File: appkotlin/src/main/java/com/onelogin/mfa/appkotlin/factors/EmptyFragment.kt
package com.onelogin.mfa.appkotlin.factors

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import com.onelogin.mfa.appkotlin.R

class EmptyFragment : Fragment() {

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        return inflater.inflate(R.layout.fragment_empty, container, false)
    }
}